<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profesori</class>
 <widget class="QDialog" name="Profesori">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profesori</string>
  </property>
  <widget class="QLabel" name="labelSifra">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Sifra</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIme">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ime</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezime">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Prezime</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDatumRodjenja">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Datum rodjenja</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTelefon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefon</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSifra">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIme">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrezime">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTelefon">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelListaUnetihProfesora">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Lista unetih profesora</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonNovi">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Novi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUpisi">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>65</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Upisi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzmeni">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Izmeni</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>155</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditDatumRodjenja">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>dd / MMM / yyyy</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>360</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
